### PR TITLE
[MISC] Update all links to use HTTPS whenever possible.

### PR DESCRIPTION
**Pull request by gllmflndn**
Checked all HTTP links and updated them if they can be served as HTTPS.

### CODE REVIEW

File: src/02-common-principles.md
Comment (effigies) [inline src/02-common-principles.md:518]: While we're here, let's do a little cleanup. There's no need for "here" or colons when we're going to spell out URLs. Also we can add line breaks to keep sentences self-contained.

```suggestion
format can be found at [https://www.json.org](https://www.json.org/).
Several editors have built-in support for JSON syntax highlighting that aids
manual creation of such files.
An online editor for JSON with built-in validation is available at
```

**Review by effigies — APPROVED** [review]
Looks good. Noticed a small thing that could be easily wrapped into this PR, but feel free to reject to keep this focused.

File: src/02-common-principles.md
Comment (gllmflndn) [reply src/02-common-principles.md]: Thanks, I agree (and sorry, I stupidly copied your change over instead of using the interface here).

**Comment by gllmflndn** [comment]
The linkchecker failed because of https://talairach.org/ [SSL: CERTIFICATE_VERIFY_FAILED]. My browser is fine but wget/curl complains too: shall I revert to HTTP for that one link?

**Comment by effigies** [comment]
Looks like a badly configured server (doesn't bundle the intermediate cert; browsers will fetch it, CURL won't): https://www.ssllabs.com/ssltest/analyze.html?d=talairach.org&latest

It's the same cert for https://brainmap.org, so possibly @angielaird would know who to contact about fixing their config?

File: src/04-modality-specific-files/02-magnetoencephalography.md
Comment (effigies) [inline src/04-modality-specific-files/02-magnetoencephalography.md:103]:
```suggestion
| CogAtlasID             | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) term that describes the task (for example, Resting State with eyes closed "[https://www.cognitiveatlas.org/task/id/trm_54e69c642d89b](https://www.cognitiveatlas.org/task/id/trm_54e69c642d89b)"). |
```

File: src/99-appendices/02-licenses.md
Comment (effigies) [inline src/99-appendices/02-licenses.md:15]:
```suggestion
| CC0        | Creative Commons Zero 1.0 Universal.                   | Use this if you are a holder of copyright or database rights, and you wish to waive all your interests in your work worldwide: [https://creativecommons.org/publicdomain/zero/1.0/](https://creativecommons.org/publicdomain/zero/1.0/) |
```

**Review by effigies — COMMENTED** [review]
A couple changes. Also, to satisfy the linter, you're going to need to align the table fences (`|`) in tables where the number of characters in a row has changed.

File: src/99-appendices/08-coordinate-systems.md
Comment (effigies) [inline src/99-appendices/08-coordinate-systems.md:144]: I've reached out to someone at the hosting institution to fix their TLS configuration, but no reason to hold this up for that.

```suggestion
| Talairach                           | Piecewise linear scaling of the brain is implemented as described in TT88. [http://talairach.org/](http://talairach.org/)                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment (gllmflndn) [reply src/99-appendices/08-coordinate-systems.md:144]: Thanks for doing the extra mile!
I wonder if the linkchecker could display a warning if the HTTP status code is not 200 so that redirections can be flagged for possible changes.

**Comment by sappelhoff** [comment]
I pushed a commit fixing the table pipe errors.

I also started using a different link style in the "licenses" appendix. All text and links themselves stayed the same, but the link is now embedded in the name of the license.

Overhauling the licenses also made me aware of our definition of the "public domain" license, which is not congruent with the information [wiki.creativecommons](https://wiki.creativecommons.org/wiki/public_domain)

We say:

> No license required for any purpose; the work is not subject to copyright in any jurisdiction.

They say:

>Works that are in the public domain in one legal jurisdiction are not necessarily in the public domain worldwide.

Somebody should take a look in a separate issue.

**Review by sappelhoff — APPROVED** [review]
LGTM generally if all links continue to work. Thanks Guillaume.

**Comment by sappelhoff** [comment]
mmh connection timeout on https://www.nitrc.org/projects/pediatricatlas ... remaining links seem okay. And the connection timeout happens for both https and http.

Maybe we can wait for a bit and it'll solve itself.

**Comment by sappelhoff** [comment]
Thanks @gllmflndn

**Pull request merged**
